### ISSUE TITLE
Voilà timeout time per cell

### ISSUE THREAD

**Issue by CasperWA** [CasperWA/voila-optimade-client#40]
If a notebook cell takes longer than 30 s to render (which will be the case if hitting an invalid URL when retrieving providers) it will automatically fail the complete render.

This can be set by starting Voilà using:

```shell
voila --VoilaExecutePreprocessor.timeout=<some_int>
```

It may also be possible to set this in a Jupyter config file. There are mentions of it in the GitHub issue of the Voilà repository, but not something explicit in their documentation.

**Comment by CasperWA** [comment]
Using Binder a JSON file `jupyter_config.json` can be placed in the repository root (see [the Voilà docs](https://voila.readthedocs.io/en/latest/deploy.html#customizing-voila-on-binder)).
Locally, a `voila.json` file (same content as for the `jupyter_config.json` file) can be placed in the Jupyter config dir, which can be found by typing `jupyter --config-dir`.

**Comment by CasperWA** [comment]
Note: To see the full set of configurations, type:
```shell
voila --help-all
```

**Comment by CasperWA** [comment]
Fixed by #43
